package.json

```diff
@@ -50,6 +50,7 @@
     "prop-types": "^15.6.2",
     "react": "^16.10.2",
     "react-autosize-textarea": "^7.0.0",
+    "react-beautiful-dnd": "^12.2.0",
```

Comment:
I looked at package-lock.json and this library looks pretty big. It requires e.g. whole redux. I googled a bit and found [another library](https://www.npmjs.com/package/react-dnd). It looks updated and has over 12k stars so it has similar popularity. Here is a demo for sortable list: https://react-dnd.github.io/react-dnd/examples/sortable/simple
It also provides custom hooks so it may work nicely with our codebase. What do you think?